src/envoy/mixer/http_filter.cc

```diff
@@ -149,6 +185,24 @@ class Instance : public Http::StreamFilter, public Http::AccessLog::Instance {
   FilterHeadersStatus decodeHeaders(HeaderMap& headers,
                                     bool end_stream) override {
     Log().debug("Called Mixer::Instance : {}", __func__);
+
+    // forward attributes (off by default)
+    if (!config_->forward_attributes().empty()) {
+      // detect disable switch in the opaque config
+      auto route = decoder_callbacks_->route()->routeEntry();
```

Comment:
If you cache it in Config, and with route pointer as key,   then you don't care which one is called first.
If cache miss, check the config json,  otherwise, use cache result.